docs/DIOXUS_BEST_PRACTICES.md

```
@@ -0,0 +1,582 @@
+cargo clippy      # Rust Linting
+```
+
+**重要**: Dioxusでは`dx bundle --release`が本番デプロイ用コマンド
+
+---
+
+## プロジェクト構造
+
+### 推奨ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)